web/public/intercom/identity.js

```diff
@@ -28,6 +28,21 @@ if (typeof userUuid === "undefined") {
   userUuid = undefined;
 }
 
+var userLegalStructure = document.currentScript.getAttribute("legal-structure");
```

Comment:
I think let tends to be better than var because its scope is narrower which leads to fewer potential weird scoping override errors.

Reply:
That's fair, but I think we should make a separate refactoring chore for this. Here AnJu is just following the pattern of the whole rest of the file. If we change it, we should change it in all places here.

Reply:
Or idk, if AnJu addresses it now, that's also fine but we should change all the instances of `var` in this whole file